nexus/src/app/oximeter.rs

```diff
@@ -127,20 +129,6 @@ impl super::Nexus {
             .db_datastore
             .producer_endpoint_upsert_and_assign(opctx, &producer_info)
             .await?;
-
-        let address = SocketAddr::from((
-            collector_info.ip.ip(),
-            collector_info.port.try_into().unwrap(),
-        ));
-        let collector =
-            build_oximeter_client(&self.log, &collector_info.id, address);
-
-        collector
-            .producers_post(&oximeter_client::types::ProducerEndpoint::from(
-                &producer_info,
-            ))
-            .await
-            .map_err(Error::from)?;
```

Comment:
I'm not sure we talked about this, but this seems like a pretty big upside of this change: if `oximeter` itself is down or misbehaving, prior to this PR, new producers would get a failure from Nexus when registering because of this post. Now producers registering is independent of the status of oximeter itself. Seems great!